docs/best_practices.md

```diff
@@ -183,7 +183,7 @@ Some top-level notes:
 - Enzyme is a unit testing library that matches React's internals 1:1. `@testing-library/react` is fundamentally a light weight html-based integration testing library. Depending on what you want to test one might be better suited than the other. Use what you think works best!
 - When testing React Hooks, [`@testing-library/react-hooks`](https://www.npmjs.com/package/@testing-library/react-hooks) is the best way to test that level of the stack.
 - We avoid snapshot tests; they produce too much churn for too little value.
-- We use the [`relay-test-utils`](https://github.com/facebook/relay/tree/main/packages/relay-test-utils) package for testing Relay code, and [this helper](https://github.com/artsy/force/blob/0b291f005763e7c2600a5077786c9510bf655079/src/v2/DevTools/setupTestWrapper.tsx) for quickly spinning up tests. There are two versions, one for Enzyme, and one for RTL. Note that this helper can't test `QueryRenderer`s; extract the render code into a fragment-like container and test that. (See the [`RegisterButton` component](https://github.com/artsy/force/blob/main/src/v2/Apps/Auction/Components/AuctionDetails/__tests__/RegisterButton.jest.tsx#L1) for an example.)
+- We use the [`relay-test-utils`](https://github.com/facebook/relay/tree/main/packages/relay-test-utils) package for testing Relay code, and [this helper](https://github.com/artsy/force/blob/0b291f005763e7c2600a5077786c9510bf655079/src/v2/DevTools/setupTestWrapper.tsx) for quickly spinning up tests. There are two versions, one for Enzyme, and one for RTL. Note that this helper can't test `QueryRenderer`s; extract the render code into a fragment-like container and test that.
```

Comment:
```
(See the [`RegisterButton` component](https://github.com/artsy/force/blob/main/src/v2/Apps/Auction/Components/__tests__/RegisterButton.jest.tsx) for an example.)
```

@jacobherrington - if a file appears missing, the Go To File button works really well :) Link updated above. 

<img width="840" alt="Screen Shot 2022-05-24 at 9 51 15 AM" src="https://user-images.githubusercontent.com/236943/170090100-245abee1-28a1-4e35-8555-d04e3a25569f.png">